awsiot/iotidentity.py

```diff
@@ -259,9 +259,10 @@ def subscribe_to_register_thing_rejected(self, request, qos, callback):
 class CreateCertificateFromCsrRequest(awsiot.ModeledClass):
     __slots__ = ['certificate_signing_request']
 
-    def __init__(self, certificate_signing_request=None):
-        # type: (typing.Optional[str]) -> None
-        self.certificate_signing_request = certificate_signing_request # type: typing.Optional[str]
+    def __init__(self, *args, **kwargs):
+        self.certificate_signing_request = kwargs.get('certificate_signing_request')
+        for key, val in zip(['certificate_signing_request'], args):
```

Comment:
```zip(self.__slots__, ...)```?

Reply:
We don't want to do slots because sometimes the slots and the list are not the same.  See GetShadowRequest/Response, DeleteShadowRequest/Response